### PR TITLE
Fixed newline problems

### PR DESCRIPTION
**Pull request by adamtherookie**

### CODE REVIEW

File: kernel/kernel/kernel.c [hunk 1]
```diff
@@ -4,5 +4,5 @@
 
 void kernel_main(void) {
 	terminal_initialize();
-	printf("Hello, kernel World!\nMade by  Ayman0x03 & aa2006\n");
+	printf("Hello, kernel World!\nMade by aa2006 & Ayman0x03");
```
Comment (Ayman0x03) [inline kernel/kernel/kernel.c:7]: You didn't like the first one ? huh ?

File: kernel/arch/i386/tty.c [hunk 2]
```diff
@@ -39,18 +39,18 @@ void terminal_putentryat(unsigned char c, uint8_t color, size_t x, size_t y) {
 }
 
 void terminal_putchar(char c) {
-	unsigned char uc = c;
-	terminal_putentryat(uc, terminal_color, terminal_column, terminal_row);
-	if (++terminal_column == VGA_WIDTH) {
-		terminal_column = 0;
-		terminal_row ++;
-	} else if (c == '\n') {
+	if (c == '\n') {
```
Comment (Ayman0x03) [inline kernel/arch/i386/tty.c:42]: Comment with a screenshot from virtualbox showing the improvements.

**Review by Ayman0x03 — CHANGES_REQUESTED** [review]
Change only the tty.c file !

File: kernel/kernel/kernel.c (hunk 1 above)
Comment (adamtherookie) [reply kernel/kernel/kernel.c:7]: >D lol

Comment (adamtherookie) [reply kernel/kernel/kernel.c:7]: What's wrong with it?

**Pull request closed without merging**